.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+
+[Makefile]
+indent_style = tab
```

Comment:
The mixture of spaces and tabs is a bit odd, perhaps we should consistently use either tabs or spaces?